skills/dff_knowledge_prompted_skill/Dockerfile

```diff
@@ -37,4 +37,7 @@ ENV ENVVARS_TO_SEND ${ENVVARS_TO_SEND}
 ARG USE_KG_DATA
 ENV USE_KG_DATA=$USE_KG_DATA
 
+ARG USE_BOT_KG_DATA
```

Comment:
```suggestion
ARG USE_BOT_KG_DATA=0
```

Reply:
otherwise in the server.py getting these vars from env will get you None values (not 0 as you pointed out in deault)

Reply:
Thanks for suggestion. Added default value to Dockerfile.